js/src/dropdown.js

```diff
@@ -159,6 +161,10 @@ const Dropdown = (($) => {
             element = parent
           }
         }
+        // If boundary is not `scrollParent`, then set position to `static` to allow the
+        // menu to "escape" the scroll parent's boundaries
+        // https://github.com/twbs/bootstrap/issues/24251
+        $(parent).css('position', this._config.boundary !== 'scrollParent' ? 'static' : '')
```

Comment:
I think this would be better.

```js
if (this._config.boundary !== 'scrollParent') {
  $(parent).css('position', 'static')
}
```

Reply:
agree with @XhmikosR :+1:

Reply:
should we prefer using inline CSS instead of using one of our CSS classes ? (`.position-static` https://getbootstrap.com/docs/4.0/utilities/position/#common-values)

Reply:
Oooh, i forgot that the positioning utilities has a .`position-static`.  it would be better than using an inline style.